lib/widget/button.dart

```diff
@@ -0,0 +1,39 @@
+import 'package:flutter/material.dart';
+
+class CustomButton extends StatelessWidget {
+  final String text;
+  final VoidCallback onPressed;
+  final Color color;
+  final Color textColor;
+  final double width;
+  final double height;
+
+  CustomButton({
+    required this.text,
+    required this.onPressed,
+    this.color = const Color(0xFFFF7622),
+    this.textColor = Colors.white,
```

Comment:
kendini Color verme. Biz yeni theme içinde var import alman gerekiyor

this.color = const Color(0xFFFF7622) olan
"Color(0xFFFF7622)" yerindeki  "primary" yazman lazım

this.color = const primary, bu olması gereken sen önce import alman gereken theme